oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -921,4 +928,26 @@ private String getReauthenticateClaim(AuthenticationRequest authenticationReques
 
         return sub;
     }
+
+    private void logBrowserSessionIdMetrics(
+            Optional<String> browserSessionIdFromSession,
+            Optional<String> browserSessionIdFromCookie) {
+        if (configurationService.isBrowserSessionCookieEnabled()) {
```

Comment:
a few nested if statements - not sure if this can be improved but not to bothered as it will be removed.